src/library-authoring/empty-page/index.scss

```diff
@@ -1,58 +1,31 @@
-.empty {
-  &-sheet-wrapper {
-    margin: 0;
-    padding: 0;
-    position: relative;
-    background: $light-200;
-  }
-
-  &-content {
-    @include clearfix();
+.empty-sheet-wrapper {
+  background: $light-200;
+}
 
-    margin: 0 auto;
-    display: flex;
-    align-items: center;
-    text-align: center;
-    flex-direction: column;
-    border-radius: ($baseline/3.3);
-    padding: ($baseline*2) 0;
-    width: flex-grid(9);
+.empty-content {
+  margin: 0 auto;
+  text-align: center;
+  padding: 3.25rem 2.5rem;
+  width: flex-grid(9);
+  font-size: 1.125rem;
```

Comment:
please clarify my, we no longer use `%font-size` mixin?

Reply:
We don't use mixins and variables out of the `studio.scss` file.

Reply:
And it's not necessary here